packages/shared/src/db/queries.ts

```diff
@@ -101,10 +104,20 @@ export const getSettings = createReadQuery(
   ['settings']
 );
 
+export const getGroupPreviews = createReadQuery(
+  'getGroupPreviews',
+  async (groupIds: string[]) => {
+    return client.query.groups.findMany({
+      where: inArray($groups.id, groupIds),
+    });
+  },
+  []
+);
+
 export const getGroups = createReadQuery(
   'getGroups',
   async ({
-    includeUnjoined,
+    includeUnjoined = false,
```

Comment:
Does this do anything?

Reply:
From Alec's work, prevents unjoined gangs from showing up in the query. Seems defensive. 

Whether it's useful or not will be more clear when I tackle the group reference issue after this lands.

Reply:
Oh I see what you mean, guessing he was just trying to make it more explicit. Removed the default.